request-schemas.ts

```diff
@@ -0,0 +1,60 @@
+import Joi = require('@hapi/joi');
+
+const creationSchema = Joi.object({
+    name: Joi.string()
+        .alphanum()
```

Comment:
can name contain ` ` /`-`? description?

Reply:
The name should be alpha-numeric. You are right about the description (removed alphanum)